lib/mayaHydra/flowViewportAPIExamples/footPrintNode/mhFootPrintNode.cpp

```diff
@@ -552,19 +598,30 @@ void* MhFootPrint::creator()
 
 void MhFootPrint::addedToModelCb()
 {
-    static const SdfPath noPrefix = SdfPath::AbsoluteRootPath();
+    _pathPrefix = SdfPath(TfStringPrintf("/MhFootPrint_%p", this));
```

Comment:
Lack of path prefix meant that the prefix was the Hydra scene index root.  Thus, when foot print node was selected, the Hydra scene index root was selected, and all its children received selection highlighting, which is the normal and correct way to highlight children of a selected prim.  Unfortunately, this meant that the whole scene was selection highlighted...  Fixed with a proper Hydra scene index prefix.